code/modules/fallout/obj/door_keys.dm

```diff
@@ -89,6 +89,7 @@
 	w_class = WEIGHT_CLASS_TINY
 	layer = 100
 	var/open = FALSE
+	var/locked = TRUE
```

Comment:
is this use anywhere? I've been drinking so I can't say for certain - but I can't find anywhere this is used.